webapp/web/selector.css

```diff
@@ -1,5 +1,4 @@
-#project-selector select {
-    min-width: 310px;
+#project-selector select, .period-filter {
```

Comment:
nit: I'd prefer to have the selectors in a list on separate lines, it helps with readability since space has meaning in CSS selectors, and commas look very much like dots if you're not looking very closely.

Reply:
Put it in a separate file